### PR TITLE
Update code to link training biomass dataset to landsat 

### PR DESCRIPTION
**Pull request by tcchiao**

### CODE REVIEW

File: carbonplan_trace/v1/load.py [hunk 1]
```diff
@@ -146,6 +144,6 @@ def biomass(tiles, year):
             complete_df = pd.concat([complete_df, df], axis=0)
         else:
             complete_df = df
-
-    complete_df = complete_df[complete_df.datetime.dt.year == year]
+    complete_df['year'] = complete_df.apply(grab_year, axis=1)
```
Comment (tcchiao) [inline carbonplan_trace/v1/load.py:147]: I didn't end up saving `datetime` in the biomass dataset despite what I commented in the other PR... reverting to Ori's previous code

File: carbonplan_trace/v1/load.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def aster(ds, tiles, lat_lon_box=None, dtype='int16'):
 
 
 def worldclim(ds, dtype='int16'):
-    mapper = fsspec.get_mapper(
+    mapper = fs.get_mapper(
```
Comment (orianac) [inline carbonplan_trace/v1/load.py:56]: i still get so confused about fsspec vs fs 😆

File: carbonplan_trace/v1/load.py (hunk 1 above)
Comment (orianac) [reply carbonplan_trace/v1/load.py:147]: yet one more +1 for version control! being able to grab previous commits!

File: notebooks/training_prep.ipynb [hunk 3]
```diff
@@ -321,16 +315,20 @@
...
-    "                    prep_training_dataset(\n",
-    "                        path=path,\n",
-    "                        row=row,\n",
-    "                        year=year,\n",
-    "                        access_key_id=access_key_id,\n",
-    "                        secret_access_key=secret_access_key,\n",
-    "                        training_write_bucket=f\"{bucket}/training/\",\n",
-    "                    )\n",
-    "                )"
+    "                scene_stores = fs.ls(landsat_bucket.format(year, path, row))\n",
```
Comment (orianac) [inline notebooks/training_prep.ipynb:318]: the only thing this makes me think is whether we'd want to calculate an `active_row_path_locations` list or something ahead of time and then just loop through those? i guess it makes no difference to do it as part of the task though!

**Pull request merged**